pyartcd/pyartcd/pipelines/build_sync.py

```diff
@@ -32,10 +34,58 @@ def __init__(self, runtime: Runtime, version: str, assembly: str, publish: bool,
         self.images = images
         self.exclude_arches = [] if not exclude_arches else exclude_arches.replace(',', ' ').split()
         self.skip_multiarch_payload = skip_multiarch_payload
+        self.triggered_from_gen_assembly = triggered_from_gen_assembly
+        self.build_number = build_number
         self.logger = runtime.logger
         self.working_dir = self.runtime.working_dir
 
+    async def comment_on_pr(self):
```

Comment:
This is useful to let the world know somebody is already syncing content from the new assembly definition. What would also help, would be to post the build-sync result to the PR once it's done. This is usually enough to justify why such a PR was merged, and we are currently doing that by hand every time. This part is easy, and I would recommend including it in this PR.

To extend this feature, we could possibly have `build-sync` find a related PR when triggered on a named assembly, and post its result there regardless of having being triggered by `gen-assembly`. This would cover such a scenario:
- gen-assembly runs, and triggers build-sync
- build-sync runs, then fails and post its outcome on the PR
- release artists fix assembly definition, build status, or whatever's needed, and trigger a new build-sync
- build-sync succeeds, and post its result on the PR
- release artists can safely merge this PR with evidence that shows it was the right thing to do
This part is not trivial, and should probably become its own ticket.

Reply:
Will create a new PR for this feature.